File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/executing/AliasRefEvaluator.java
package org.opencds.cqf.cql.engine.elm.executing;

import org.opencds.cqf.cql.engine.execution.State;

public class AliasRefEvaluator {
    public static Object internalEvaluate(String name, State state) {
        return state.resolveAlias(name);
    }
}

File: Src/java/engine/src/main/kotlin/org/opencds/cqf/cql/engine/elm/executing/AliasRefEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

import org.opencds.cqf.cql.engine.execution.State

object AliasRefEvaluator {
    @JvmStatic
    fun internalEvaluate(name: String?, state: State?): Any? {
        return state!!.resolveAlias(name)
    }
}
